### ISSUE TITLE
Create a wizard for updating select data tables

### ISSUE THREAD

**Issue by edwinmp** [devinit/ddw-analyst-ui#166]
- A wizard that operates similarly to https://app.datawrapper.de/
- Initially accepts CSV & Excel as inputs
- [Low Priority] accepts API as input

**Comment by edwinmp** [comment]
Currently waiting on the backend dependency in issue #165
